core/controllers/library.py

```diff
@@ -143,6 +148,78 @@ def get(self):
         self.render_json(self.values)
 
 
+class LibraryGroupPage(base.BaseHandler):
+    """The page for displaying top rated and recently published explorations.
+    """
+
+    def get(self):
+        """Handles GET requests."""
+        group_mode = True
```

Comment:
No need to declare this separately, just substitute group_mode for True in the return dict.